app/assets/javascripts/attraction_list.js

```diff
@@ -0,0 +1,7 @@
+// $(document).ready(function(){
+  var loadAttractionList = function(attraction, marker){
+
+    var attractionInfo = ['<li id="attraction', attraction._id + '">',attraction.name,'</li>'].join('')
```

Comment:
Take a look at [Underscore.js templates](http://underscorejs.org/#template) for a bit of a nicer way to handle this.